sdcs-site/src/app/channel/channel/channel.component.ts

```diff
@@ -1,15 +1,77 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, Input } from '@angular/core';
+import { NgForm, FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import { ChannelService } from './../../services/channel/channel.service';
```

Comment:
Just an observation question - is there a better way to include services than putting a relative path (../../../)?